checkQC/handlers/qc_handler.py

```diff
@@ -248,3 +252,14 @@ def report(self):
 
         return sorted_errors_and_warnings
 
+    def downgrade_errors(self, errors_and_warnings):
+        """
+        Takes a list of error and warnings and downgrade all errors to warnings.
```

Comment:
Perhaps it would be clearer to have the downgrading as an option on the handler, and then pass it in when it is instantiated? This way you wouldn't need to do any extra parsing under `_compile reports`, but you could handle all the downgrading in the `QCHandler`. What do you think?

Reply:
I guess you could then pass it along in the class configuration?